### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [gabrielsmorais/f1-3-c2p1-colmar-academy#4]
Every image should have an alt text that gives a description, screen readers read that information to the visually impaired so they can navigate around your site better.
